### ISSUE TITLE
Cannot reproduce the model performance in cylinder

### ISSUE THREAD

**Issue by wuhaixu2016** [OrchardLANL/Senseiver#6]
First of all, thanks for this clean code, which is easy to read and run.

However, I feel weird that when reproducing the code with the default script in README.md, the performance on the cylinder is extremely bad. The training process early stops at around 400 epochs and the predicted result is a noise.

I am wondering if the training epoch is not enough. To ensure reproducibility, I think it is necessary to provide more training details, such as the convergence epoch number of each dataset.

**Comment by wuhaixu2016** [comment]
In addition, did you remove the zero points when calculating the relative L2?

**Comment by jmsweng** [comment]
I was able to reproduce the cylinder results but had to train for more epochs, around 1000 usually seemed to give good results but the number of epochs required also varied depending on the specific GPU I used as well. Occasionally I would get results that mostly contained streaky artifacts, I suspect because the training got stuck in some local minimum, so the learning rate parameter might need some tuning as well.
